packages/server/graphql/intranetGraphQLHandler.ts

```diff
@@ -35,7 +35,7 @@ const intranetHttpGraphQLHandler = uWSAsyncHandler(async (res: HttpResponse, req
       ip,
       query,
       variables,
-      isPrivate,
+      isPrivate: true,
```

Comment:
If we treat everything coming to intranet-graphql as isPrivate, we can avoid the problems with setting the property manually and use the GraphQL resource type from retool.